test/integration/api/audits.js

```diff
@@ -636,9 +636,9 @@ describe('/audits', () => {
         .then(({ body }) => {
           body.length.should.equal(4);
           body.map(a => a.action).should.eql([
+            'dataset.create',
```

Comment:
Not quite sure why this is after form publish when it happens within form publish.

Reply:
it had to do with not using `publish.audit.withResult`, which made the logging happen earlier, i think.